content/zh-cn/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -36,40 +36,42 @@ Kubespray 提供：
   - Flatcar Container Linux by Kinvolk
   - Debian Bullseye, Buster, Jessie, Stretch
   - Ubuntu 16.04, 18.04, 20.04, 22.04
-  - CentOS/RHEL 7, 8
-  - Fedora 34, 35
+  - CentOS/RHEL 7, 8, 9
+  - Fedora 35, 36
   - Fedora CoreOS
   - openSUSE Leap 15.x/Tumbleweed
-  - Oracle Linux 7, 8
-  - Alma Linux 8
-  - Rocky Linux 8
-  - Amazon Linux 2 
+  - Oracle Linux 7, 8, 9
+  - Alma Linux 8, 9
+  - Rocky Linux 8, 9
+  - Kylin Linux Advanced Server V10
+  - Amazon Linux 2
 * Continuous integration tests.
 -->
-* 高可用性集群
-* 可组合属性（例如可选择网络插件）
-* 支持大多数流行的 Linux 发行版
-   * Kinvolk 的 Flatcar Container Linux
-   * Debian Bullseye、Buster、Jessie、Stretch
-   * Ubuntu 16.04、18.04、20.04, 22.04
-   * CentOS/RHEL 7、8
-   * Fedora 34、35
-   * Fedora CoreOS
-   * openSUSE Leap 15.x/Tumbleweed
-   * Oracle Linux 7、8
-   * Alma Linux 8
-   * Rocky Linux 8
-   * Amazon Linux 2
-* 持续集成测试
+- 高可用性集群
+- 可组合属性（例如可选择网络插件）
+- 支持大多数流行的 Linux 发行版
+  - Flatcar Container Linux by Kinvolk
```

Comment:
Not sure should we translate this or not. Maybe just remove the maintainer name.
```suggestion
  - Flatcar Container Linux
```